features/step_definitions/basic_steps.rb

```diff
@@ -1,25 +1,13 @@
 Given(/^I'm on the "([^"]*)" page$/) do |page|
-  case page
-    when 'home'
-      visit '/'
-    when 'Sign up'
-      visit 'users/sign_up'
-  end
+  visit root_path
```

Comment:
This is now what I meant. Now whatever page you want to visit you will navigate to the `root_path` Tha case statement was all good. I just wanted you to use path helpers. Do `rails routes` in your terminal and you will find the path for 'users/sign_up'. The just add `_path` at the end and use it in your case.